imagine/priors/prior.py

```diff
@@ -184,7 +184,7 @@ def __init__(self, samples=None, pdf_fun=None, xmin=None, xmax=None,
 
         # Evaluates the PDF
         pdf_x = np.linspace(xmin_val, xmax_val, pdf_npoints)
-        pdf_y = pdf_fun(pdf_x << unit)
```

Comment:
I disagree with this change: for consistency, the supplied pdf_fun should be able to operate on Quantities instead of pure floats.  The reason for this is to trap the situation where someone has a prior where the highest probability is at, say 1 tesla, and someone uses it as if the highest probability is 1 gauss. 

That said, there is an error in that line:  in the case where the parameter is *dimensionless*, `unit` is `None` and the code will break. I will amend this right now.

Reply:
see 0a3c995

Reply:
I agree with you, this was just a hotfix in order to get stuff to run for me. I should have stressed that.